From 78619b739effe2f2b5a7cd1f6f024a6c7f78387d Mon Sep 17 00:00:00 2001
From: Christian Hohnstaedt <Christian.Hohnstaedt@wago.com>
Date: Wed, 13 Nov 2019 13:00:46 +0100
Subject: [PATCH] edma: Add missing newline in debug output

and improve the center calculation of the header.
No functional change

Signed-off-by: Christian Hohnstaedt <Christian.Hohnstaedt@wago.com>
---
 drivers/dma/edma.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/dma/edma.c b/drivers/dma/edma.c
index cf95eca..45facf6 100644
--- a/drivers/dma/edma.c
+++ b/drivers/dma/edma.c
@@ -889,11 +889,11 @@ static void dfs_print_table(struct seq_file *m, int cnt, const char **bufs,
 			if (q-p > lengths[i])
 				lengths[i] = q-p;
 		}
-		len += lengths[i];
+		len += lengths[i] + 3;
 	}
 	capt_offs = (len - capt_len) / 2 + capt_len;
 	if (capt)
-		seq_printf(m, "%*s\n%*.*s", capt_offs, capt,
+		seq_printf(m, "%*s\n%*.*s\n", capt_offs, capt,
 				capt_offs, capt_len,
 				"-------------------------------------");
 	while (1) {
-- 
2.7.4

